iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/v1/models/DeviceRegistryApiModel.java

```diff
@@ -28,7 +30,7 @@
     private DeviceTwinProperties properties;
     private boolean isSimulated;
 
-    private final String dateFormatString = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
+    private final String dateFormatString = "yyyy-MM-dd'T'HH:mm:ss:SSS'Z'";
```

Comment:
There are other places which are using dot instead of colon. Are you going to change them all?